src/hooks/useEndpointsConfig.ts

```diff
@@ -24,6 +24,6 @@ export const useEndpointsConfig = () => {
     validators: endpointsConfig.validators,
     '0xsquid': endpointsConfig['0xsquid'],
     nobleValidator: endpointsConfig.nobleValidator,
-    faucet: endpointsConfig['faucet'],
+    faucet: endpointsConfig.faucet,
```

Comment:
outta curiosity, why doesn't line 25 get changed to `endpointsConfig.0xsquid`?

Reply:
I think because it doesn't start with a letter